acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -180,3 +180,42 @@ function acquia_lift_profiles_update_7002(&$sandbox) {
     return t('All actions were synchronized to Lift Web');
   }
 }
+
+/**
+ * Convert existing vocabulary mappings to use taxonomy context module.
+ */
+function acquia_lift_profiles_update_7003() {
+  // 1) Convert existing mappings (if not already).
+  //
+  // Example
+  // * Before migration:
+  //   array (
+  //    'content_section' => '2',
+  //    'content_keywords' => '',
+  //    'persona' => 'taxonomy_context__country',
```

Comment:
As discussed on Slack, [updated](https://github.com/acquia/acquia_lift/commit/64539a26715b7c38e8b24d649b43867dbe2d2d30) to not to mention the "out of norm" case.